.github/workflows/bump_version.yml

```
@@ -0,0 +1,79 @@
+name: Bump Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      newVersion:
+        description: "New version"
+        required: true
+        type: string
+
+env:
+  PHOENIX_VERSION: 1.6.6
+  MIX_ENV: test
+
+jobs:
+  release_version_test:
+    name: Bump version
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Setup asdf
+        uses: asdf-vm/actions/setup@v1
+
+      - name: Cache asdf
+        uses: actions/cache@v3
+        with:
+          path: /home/runner/.asdf
+          key: ${{ runner.os }}-asdf-${{ hashFiles('**/.tool-versions') }}
+          restore-keys: ${{ runner.os }}-asdf-
+
+      - name: Install dependencies in .tool-versions
+        uses: asdf-vm/actions/install@v1
+
+      - name: Install rebar
+        run: mix local.rebar --force
+
+      - name: Install hex
+        run: mix local.hex --force
+
+      - name: Cache Elixir build
+        uses: actions/cache@v3
+        with:
+          path: |
+            _build
+            deps
+          key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
+          restore-keys: ${{ runner.os }}-mix-
+
+      - name: Install Dependencies
+        run: mix deps.get
+
+      - name: Compile dependencies
+        run: mix compile --warnings-as-errors --all-warnings
+
+      - name: Install Phoenix ${{ env.PHOENIX_VERSION }}
+        run: make install_phoenix PHOENIX_VERSION=${{ env.PHOENIX_VERSION }}
+
+      - name: Bump version
+        run: mix nimble_template.bump_version ${{ github.event.inputs.newVersion }}
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          token: ${{ secrets.WIKI_ACTION_TOKEN }}
+          commit-message: Bump to ${{ github.event.inputs.newVersion }}
+          committer: Dev Nimble <dev@nimblehq.co>
```

We should use the bot user: 
```suggestion
          committer: Nimble Bot <bot@nimblehq.co>
```

---

adjusted it in https://github.com/nimblehq/elixir-templates/pull/219/commits/8a0f97488fdf4bc0da852e8eeab73f737e256c55, we can see it on https://github.com/nimblehq/elixir-templates/pull/224